plugins/sonar-batch-maven-plugin/src/main/java/org/sonar/plugins/batch/maven/MavenProjectBuilder.java

```diff
@@ -51,7 +51,8 @@ public void build(Context context) {
   private void setMavenProjectIfApplicable(ProjectDefinition definition) {
     if (mavenSession != null) {
       String moduleKey = definition.getKey();
-      for (MavenProject mavenModule : (List<MavenProject>) mavenSession.getSortedProjects()) {
+      for (MavenProject mavenModule : (List<MavenProject>) mavenSession.getProjects()) {
+        // FIXME assumption that moduleKey was not modified by user and follow convention <groupId>:<artifactId>
```

Comment:
:new: ![MAJOR](https://raw.githubusercontent.com/henryju/image-hosting/master/major.png)Take the required action to fix the issue indicated by this comment.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1134)